multicast/src/test/kotlin/com/dropbox/flow/multicast/ChannelManagerTest.kt

```diff
@@ -83,7 +83,7 @@ class ChannelManagerTest {
             }
             upstream.close(TestException())
             collection.await()
-            fail("collection should propagate upstream exception.")
+            throw AssertionError("collection should propagate upstream exception.")
```

Comment:
We can actually keep it as is by just importing `org.junit.Assert.fail`

Reply:
@yigit - curious, does google/Truth have a philosophical objection to a `fail` function or is it an oversight in v1 public release?

Reply:
i have no idea, it was just not available so i replaced w/ an assertion error.